### ISSUE TITLE
no works DP, NO DUAL MONITOR

### ISSUE THREAD

**Issue by GerardCHC** [hackintosh-efi/dell-optiplex-3080-mff#9]
Hello i apreciated your work, the efi works apropiatly but no have dualmonitor because DISPLAYPORT NO WORKS

**Comment by mrasong** [comment]
You may use a DP to HDMI hub?

**Comment by GerardCHC** [comment]
Yes i use DP to HDMI HUB but no works in MacOS, i'm Windows Ok dual monitor, but in MacOS no works 🙁

**Comment by mrasong** [comment]
There are two kind of DP 2 HDMI hubs.
Active and passive.
You need an  active adapter which can give real DP signal.
The passsive one only the interface was converted, monitor got HDMI signal not DP.

The HP DP to HDMI hub maybe a better choose, which I'm using works well.

**Comment by Donan123** [comment]
When the login screen appears after booting, the DP display goes black for about 5 minutes. To avoid this, the monitor power must be turned off and on again. I am not using DP to HDMI hub, solely the DP cable to monitor.
